### PR TITLE
EXPDAT-072 About us page

### PR DESCRIPTION
**Pull request by ettha**

### CODE REVIEW

File: app/src/main/java/at/sw21_tug/team_25/expirydates/ui/home/AboutUsView.kt [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            popupWindow.elevation = 10.0F
+            popupWindow.isFocusable = true
+
+            activity.findViewById<View>(android.R.id.content).post {
+                val view = activity.findViewById<View>(android.R.id.content).rootView
+                popupWindow.showAtLocation(view, Gravity.CENTER, 0, 0)
+
+            }
+
+/*            val closePopUpButton = popupView.findViewById<Button>(R.id.closePopUp)
```
Comment (ChrisFartek) [inline app/src/main/java/at/sw21_tug/team_25/expirydates/ui/home/AboutUsView.kt]: Delete commented code?

**Review by ChrisFartek — CHANGES_REQUESTED** [review]
Auskommentierten Code entfernen

File: app/src/main/java/at/sw21_tug/team_25/expirydates/ui/home/AboutUsView.kt (hunk 1 above)
Comment (ettha) [reply app/src/main/java/at/sw21_tug/team_25/expirydates/ui/home/AboutUsView.kt]: Code has been deleted

**Review by ChrisFartek — APPROVED** [review]
Clean Code, all tests pass locally

**Pull request merged**